projects/igniteui-angular/src/lib/date-picker/date-picker.component.spec.ts

```diff
@@ -480,6 +481,61 @@ describe('IgxDatePicker', () => {
         expect(month.innerText.trim()).toBe(expectedResult.trim());
     }));
 
+    describe('Drop-down opening', () => {
```

Comment:
Do we have a test with retemplated datepicker? If not please write one.

Reply:
yes, there are already tests for retemplated dropdown and dialog date-picker.